executor/load_data.go

```diff
@@ -459,13 +459,10 @@ func (e *LoadDataInfo) indexOfTerminator(bs []byte, isInQuoter bool) int {
 		}
 	}
 	atFieldStart := true
-	inQuoter := false
 loop:
 	for i := 0; i < len(bs); i++ {
 		if atFieldStart && bs[i] == e.FieldsInfo.Enclosed {
-			if !isInQuoter {
-				inQuoter = true
-			}
+			inQuoter = !inQuoter
```

Comment:
So all cases where one string crosses multiple lines will cause an error, right?
Is it possible to verify that in the test case?

Reply:
not all cases, this error will only happen if one string is split into 2 parts and crosser multiple lines. e.g. first part `"123` second part `456\n789"` the `\n` in the second part should not interpret as new lines.

the test case  No.4 test this scenario